### PR TITLE
Merge pull request #10 bwehbe/include<stdexcept>

### PR DESCRIPTION
**Pull request by bwehbe**
missing #include<stdexcept> in files: DynamicModel.hpp , KinematicModel.hpp , ModelSimulation.hpp

### CODE REVIEW

File: src/DynamicModel.hpp [hunk 1]
```diff
@@ -2,6 +2,7 @@
 #define _DYNAMIC_MODEL_H_
 
 #include "DataTypes.hpp"
+#include <stdexcept>
```
Comment (doudou) [inline src/DynamicModel.hpp]: The header does not refer to exceptions ... What do you try to fix here ?

Comment (bwehbe) [inline src/DynamicModel.hpp]: when building I am getting an error that 'runtime_error' is not a member of 'std'

Comment (doudou) [inline src/DynamicModel.hpp]: Then the headers need to be added to the .cpp files in which the runtime_error is present, not "globally" in the headers (which impacts the library's users as well).

**Comment by bwehbe** [comment]
#include <stdexcept> moved to the source files. But note that this library is included originally in the RK4Integrator.hpp file

**Comment by doudou** [comment]
> But note that this library is included originally in the RK4Integrator.hpp file

Good catch ... would you mind moving it to the .cpp as well while you're at it ? ;-)

**Comment by bwehbe** [comment]
sure, done!

**Comment by doudou** [comment]
Thanks !

**Pull request merged**